### DISCUSSION TITLE
How long do I have to wait for support to answer my reinstatement request?

### DISCUSSION THREAD

**Discussion by schlagmichdoch** [community/community#111218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account got flagged as spam by GitHub on 17th February. The process was silent without any notification. All repositories, PR, issues and comments are shadow hidden to the public and some issues and comments were even deleted by something called 'hubot'. 

After realizing my account was flagged one day later, I created a reinstatement request ticket with the private GitHub Support. It have been waiting 18 days and I have still not received a response or any notification at all.

This is a lot longer than what the info box currently says: „ Due to high support volume, it may take up to seven business days to hear back from our Support Engineers.“ and not acceptable.

This happened although I have public repository hosting libre software with 2.3k stars and clearly am not a bot. I’m very disappointed in the whole process and consider moving the whole project to another platform.

How long will I need to wait for a response?
Ticket #2603911

Thanks in advance!

**Comment by kika1s1** [comment]
I also get the same thing, and i tried to reach them out but i did not get any reply from them

**Comment by PsVenom** [comment]
This is exactly what happened to me. It's been a whole week since I filled out the form and there has been zero response from the github team. I'm seriously disappointed with the lack of action

**Comment by laszloh2** [comment]
I am also still waiting, so more than 50 days or never.

**Reply by kika1s1** [reply]
Wait patiently, they will fix it.

**Reply by laszloh2** [reply]
In German: "dein Wort in Gottes Ohr" (basically "From your mouth to God's ears.").

**Comment by Islam412** [comment]
Response #2
